modules/oe_list_pages_link_list_source/tests/src/FunctionalJavascript/ListPageLinkSourcePluginTest.php

```diff
@@ -85,8 +90,8 @@ public function testPluginConfigurationForm(): void {
     // Edit the link list and check the values are correctly pre-populated.
     $this->drupalGet($link_list->toUrl('edit-form'));
 
-    $this->assertOptionSelected('Source entity type', 'Taxonomy term');
-    $this->assertOptionSelected('Source bundle', 'Vocabulary one');
+    $this->assertSession()->optionExists('Source entity type', 'Taxonomy term');
+    $this->assertSession()->optionExists('Source bundle', 'Vocabulary one');
```

Comment:
We have to check attribute "selected" for full and proper replacement of previous deprecated method `assertOptionSelected()`.

Reply:
This will be fixed on next update.